src/ripple/test/jtx/impl/Env.cpp

```diff
@@ -373,6 +384,46 @@ Env::submit (JTx const& jt)
 }
 
 void
+Env::sign_and_submit(JTx const& jt, Json::Value params)
+{
+    bool didApply;
+
+    auto const account =
+        lookup(jt.jv[jss::Account].asString());
+    auto const& passphrase = account.name();
+
+    Json::Value jr;
+    if (params.isObject())
+    {
+        // Use the provided parameters, and go straight
+        // to the (RPC) client.
+        if (!params.isMember(jss::secret) &&
+            !params.isMember(jss::key_type) &&
+                !params.isMember(jss::seed) &&
+                    !params.isMember(jss::seed_hex) &&
+                        !params.isMember(jss::passphrase))
+        {
+            params[jss::secret] = passphrase;
+        }
+        params[jss::tx_json] = jt.jv;
+        jr = client().invoke("submit", params);
+    }
+    else
+    {
+        // Use the command line interface
+        assert(params.isNull());
+        auto const jv = to_string(jt.jv);
```

Comment:
I am not saying to change it, but I have come to prefer `boost::lexical_cast<std::string>(jt.jv)` over the `to_string` idiom.